core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -214,6 +216,8 @@ class ByteEfficiencyAudit extends Audit {
 
     const wastedBytes = results.reduce((sum, item) => sum + item.wastedBytes, 0);
 
+    // `wastedMs` may be negative, if making the opportunity change could be detrimental.
+    // This is useful information in the LHR and should be preserved.
```

Comment:
added this comment because my first thought was "why isn't `wastedMs` being passed through a `Math.max(0, wastedMs)`?", but it's actually very useful information to know that the opportunity would be negative, so I wanted to make that explicit to future readers of the code. Happy to move/bikeshed.